python/nav/web/netmap/urls.py

```diff
@@ -74,9 +74,8 @@
         NetmapGraph.as_view(),
         name='netmap-graph-view',
     ),
-    url(r'^traffic/layer(?P<layer>[2|3])/$',
+    url(r'^traffic/layer(?P<layer>[2|3])/(?P<locationId>[\w ]+)/$',
```

Comment:
This pattern still doesn't match all valid room and location names, and is causing 404 errors on my test install.


[Here's an example of another URL pattern used to match room IDs in NAV](https://github.com/UNINETT/nav/blob/59ce9219ab135fc8c96aaf0f44ab039463d450a1/python/nav/web/info/room/urls.py#L27)